iceoryx_posh/test/moduletests/test_mepoo_handler.cpp

```diff
@@ -349,3 +349,42 @@ TEST_F(MemoryManager_test, addMemPoolWithChunkCountZeroShouldFail)
     mempoolconf.addMemPool({32, 0});
     EXPECT_DEATH({ sut->configureMemoryManager(mempoolconf, allocator, allocator); }, ".*");
 }
+
+TEST_F(MemoryManager_test, MempoolIndexOutOfRange)
+{
+    constexpr uint32_t ChunkCount{100};
+    mempoolconf.addMemPool({128, ChunkCount});
+    sut->configureMemoryManager(mempoolconf, allocator, allocator);
+
+    std::vector<iox::mepoo::SharedChunk> chunkStore;
+    for (size_t i = 0; i < ChunkCount; i++)
```

Comment:
Why are you filling the `chunkStore` here when you want just access an out of bounds index?